testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/builder/ExternalLibrariesSetupHelper.java

```diff
@@ -57,15 +56,28 @@ public void setupExternalLibraries(boolean initShippedCode) throws Exception {
 			assertTrue("Error while saving external library preference changes.", result.isOK());
 		}
 
+		Injector n4jsInjector = N4JSActivator.getInstance().getInjector("org.eclipse.n4js.N4JS");
```

Comment:
Why can't we inject the instance directly?

Reply:
Because the test runner does not support that, unfortunately.